apps/api/src/app.module.ts

```diff
@@ -58,6 +59,7 @@ const modules: Array<Type | DynamicModule | Promise<DynamicModule> | ForwardRefe
   MessagesModule,
   PartnerIntegrationsModule,
   TopicsModule,
+  getOTELModule(),
```

Comment:
it would be better if you could just return the telemetry module instead of returning the getter function that then you call like we have other modules above here